src/frontend/src/scheduler/local.rs

```diff
@@ -301,6 +301,51 @@ impl LocalQueryExecution {
                     node_body: Some(node_body),
                 })
             }
+            PlanNodeType::BatchLookupJoin => {
+                // Need to set up source templates for LookupJoinExecutor
+                let local_execute_plan = LocalExecutePlan {
+                    plan: None,
+                    epoch: self.epoch.expect(
+                        "Local execution mode has not acquired the epoch when generating the plan.",
+                    ),
+                };
+
+                // Only 1 worker is used since distribution of BatchLookupJoin is always single
+                let worker = self.front_env.worker_node_manager().next_random()?;
+
+                let exchange_source = ExchangeSource {
+                    task_output_id: Some(TaskOutputId {
+                        task_id: Some(ProstTaskId {
+                            task_id: 0,
+                            stage_id: 0, // this value doesn't matter
+                            query_id: self.query.query_id.id.clone(),
+                        }),
+                        output_id: 0,
+                    }),
+                    host: Some(worker.host.as_ref().unwrap().clone()),
```

Comment:
The host should not be determined here, it should be determined at runtime by partition prunning. 
1. For how to get the vnode(partition) of the primary key, you can find it here:
https://github.com/singularity-data/risingwave/blob/b6189745e2ee9f863a6e17b65fc134f409f65442/src/frontend/src/scheduler/plan_fragmenter.rs#L415 

2. For how to find worker node according to vnode/parallel unit, you can find it here:
https://github.com/singularity-data/risingwave/blob/b6189745e2ee9f863a6e17b65fc134f409f65442/src/frontend/src/scheduler/distributed/stage.rs#L305

Reply:
But current implementation doesn't affect  correctness, it just affects performance, so I think we can leave it now.